src/ExNavigationStack.js

```diff
@@ -435,6 +435,16 @@ class ExNavigationStack extends PureComponent<any, Props, State> {
     );
   }
 
+  _getNavigationBarHeight(latestRouteConfig) {
+    let height = NavigationBar.DEFAULT_HEIGHT;
+
+    if (latestRouteConfig.statusBar && latestRouteConfig.statusBar.translucent) {
+      height = NavigationBar.DEFAULT_HEIGHT_WITHOUT_STATUS_BAR + 24;
```

Comment:
@chirag04 Hmm... okay, it's wrong though I remember when @brentvatne changed my app to exponent he had used 24 and it felt like the header title wasn't vertically centered (since it was 1px off).

What's the issue?